packages/dd-trace/src/plugins/plugin.js

```diff
@@ -26,10 +26,23 @@ class Subscription {
 }
 
 module.exports = class Plugin {
-  constructor (tracer) {
+  constructor (tracer, tracerConfig) {
     this._subscriptions = []
     this._enabled = false
     this._tracer = tracer
+    this.config = {} // plugin-specific configuration, unset until .configure() is called
```

Comment:
This value is set later but I like having its presence be a little more explicit.